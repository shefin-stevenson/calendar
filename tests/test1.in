1
event1#
12/21 03:40
12/21 03:43
This is an event read in from the command line#
4
9
