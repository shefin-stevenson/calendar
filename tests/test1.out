

Enter a number from the list below to select an action, or press 9 to quit

Edit Commands 

1. Insert new event
2. Delete event
3. Edit event

 Print Commands

4. Print all
5. Print time block
6. Print conflicts

 Search Commands

7. Search by name
8. Search by keyword
Enter new event name (no more than 16 characters):
Enter new event start time (mm/dd hh:mm):
Enter new event end (mm/dd hh:mm):
Enter new event description (no more than 256 characters):

Enter a number from the list below to select an action, or press 9 to quit

Edit Commands 

1. Insert new event
2. Delete event
3. Edit event

 Print Commands

4. Print all
5. Print time block
6. Print conflicts

 Search Commands

7. Search by name
8. Search by keyword
event1#
12/21 3:40
12/21 3:43
This is an event read in from the command line#




Enter a number from the list below to select an action, or press 9 to quit

Edit Commands 

1. Insert new event
2. Delete event
3. Edit event

 Print Commands

4. Print all
5. Print time block
6. Print conflicts

 Search Commands

7. Search by name
8. Search by keyword
